backend/FwLite/FwLiteProjectSync/DryRunMiniLcmApi.cs

```diff
@@ -259,4 +279,69 @@ public async Task AddComplexFormType(Guid entryId, Guid complexFormTypeId)
         DryRunRecords.Add(new DryRunRecord(nameof(AddComplexFormType), $"Add complex form type {complexFormTypeId}, to entry {entryId}"));
         await Task.CompletedTask;
     }
+
+    public Task<WritingSystems> GetWritingSystems()
```

Comment:
not sure why these were all created, there should be a source generator creating passthrough methods, maybe your editor didn't realize that?